mycroft/util/lang/parse_fr.py

```diff
@@ -334,7 +334,7 @@ def number_ordinal_fr(words, i):
             strOrd = "2e"
         elif word.endswith("ième"):
             val2 = None
-            word = word[:-5]
+            word = word[:-4]
```

Comment:
Is this a bug fix or an actual change in array indexing semantics?

Reply:
Ack, thanks for clarifying!

Reply:
Should have made a more clear commit message in the first place.